### PR TITLE
Assignments 3 & 4

### PR DESCRIPTION
**Pull request by Fatima-Illana**

### CODE REVIEW

**Comment by ocorcho** [comment]
Assignment 3 ready to be accepted. @dgarijo can check assignment 4

File: Assignment4/Fátima-Illana-d21c005/FátimaIllanaTask07.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+print()
+print("SPARQL:")
+q2 = """
+PREFIX ns: <http://somewhere#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+
+SELECT DISTINCT ?individual
+WHERE {
+    ?individual a ns:Person
```
Comment (dgarijo) [inline Assignment4/Fátima-Illana-d21c005/FátimaIllanaTask07.py]: you are not listing all those instances from any subclass of person

Comment (dgarijo) [reply Assignment4/Fátima-Illana-d21c005/FátimaIllanaTask07.py]: both in rdflib and sparql

File: Assignment4/Fátima-Illana-d21c005/FátimaIllanaTask07.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+q3 = """
+PREFIX ns: <http://somewhere#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+
+SELECT ?individual ?Property ?Class ?valueProperty
+WHERE {
+    ?individual rdf:type ns:Person.
+    ?individual ?Property ?valueProperty.
+    ?Property rdf:type ?Class
+}"""
```
Comment (dgarijo) [inline Assignment4/Fátima-Illana-d21c005/FátimaIllanaTask07.py:151]: same as above

**Review by dgarijo — CHANGES_REQUESTED** [review]

**Pull request closed without merging**